users.js

```diff
@@ -397,19 +423,16 @@ class User {
 		this.send(`|popup|` + message.replace(/\n/g, '||'));
 	}
 	getIdentity(roomid) {
-		if (this.locked) {
-			return '‽' + this.name;
-		}
-		if (this.namelocked) {
-			return '‽' + this.name;
+		if (this.locked || this.namelocked) {
+			return (Config.punishgroups.locked || '‽') + this.name;
```

Comment:
mhm that is correct
https://github.com/sparkychildcharlie/Pokemon-Showdown/blob/c2c5bbd4a526eebc4852fdb51325d470ddec5b86/users.js#L206 <-- uses the group's id (aka  'locked')
https://github.com/sparkychildcharlie/Pokemon-Showdown/blob/c2c5bbd4a526eebc4852fdb51325d470ddec5b86/users.js#L221 <-- the hardcode catch does it the same way too